lib/src/rules/pub/package_descriptions.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2019, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:linter/src/analyzer.dart';
+import 'package:linter/src/utils.dart';
```

Comment:
Unused import?

FWIW: our pana baseline bot flags this (though it's noisy!):

https://travis-ci.org/dart-lang/linter/jobs/533694054